python/sysbench/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip 
+RUN curl -s https://packagecloud.io/install/repositories/akopytov/sysbench/script.rpm.sh | bash
```

Comment:
Please update this installation method to be explicit about the version of sysbench being installed.

Reply:
done